doc/content/verification_and_validation/val-2c.md

```diff
@@ -114,34 +113,125 @@ The case and model parameters used in both approaches in TMAP8 are listed in [va
+\end{equation}
+
+for HTO, and
+
+\begin{equation} \label{eq:optimization_penalty_T2}
+\delta_{\text{T}_2} = (\log(c_{\text{T}_2}^{exp}) - \log(c_{\text{T}_2}^{mod}))^2
+\end{equation}
+
+for T$_2$. The metric to be optimized is then defined as the time integral of
+
+\begin{equation} \label{eq:optimization_metric}
+g = \frac{\delta_{\text{HTO}}^2+8000}{30 \delta_{\text{HTO}}^4+400 \delta_{\text{HTO}}^2+1} + \frac{\delta_{\text{T}_2}^2+45000}{0.1 \delta_{\text{T}_2}^4+50 \delta_{\text{T}_2}^2+1}.
+\end{equation}
+
+Notably, the integral difference is defined in logarithmic space to give equal weight to all data points in the logarithmic scale during the optimization process.
+The complexity of the optimization metric is due to the large difference in scale for each species, as well as the discrete nature of the T$_2$ measurements compared to the almost continuous nature of the HTO measurements. These differences make it challenging to optimize the fits of both species.
+
+The comparison between the original and calibrated values of selected model parameters is summarized in [val-2c_parameters_calibrated].
+
+!table id=val-2c_parameters_calibrated caption=Calibrated model parameters values for the delayed injection case in val-2c..
+| Parameter       | Non-calibrated values (see [val-2c_parameters])             | Calibrated values using [Parallel Subset Simulation](https://mooseframework.inl.gov/source/samplers/ParallelSubsetSimulation.html) | Unit       |
+| --------------- | ----------------------------------------------------------- | ----------------------- | ---------- |
+| $K^0$           | $\boldsymbol{2.8} \times 2.0 \times 10^{-10}$               | 2.833 $\times 10^{-11}$ | m$^3$/Ci/s |
+| $D^e$           | 4.0 $\times 10^{-12}$                                       | 3.864 $\times 10^{-12}$ | m$^2$/s    |
+| $D^w$           | 1.0 $\times 10^{-14}$                                       | 1.737 $\times 10^{-14}$ | m$^2$/s    |
+| $K_S^e$         | $\boldsymbol{1.0 \times 10^{-3}} \times 4.0 \times 10^{19}$ | 2.514 $\times 10^{16}$  | 1/m$^3$/Pa |
+| $K_S^w$         | $\boldsymbol{3.0 \times 10^{-4}} \times 6.0 \times 10^{24}$ | 9.862 $\times 10^{20}$  | 1/m$^3$/Pa |
+| $t_{injection}$ | 10800                                                       | 9536                    | s          |
+
+## Results and Discussion
+
+!alert note title=Update from [!cite](Simon2025)
+The results presented here are updated results from thos presented in [!cite](Simon2025). First, the initial time step was reduced from dt=60 s in [!cite](Simon2025) to dt=1 s in the current case. This slightly affects the results for both the immediate and delayed injection cases. However, the results are qualitatively unchanged and conclusions remain valid. Second, the calibration approach was updated since [!cite](Simon2025) with an updated multi-objective function, and new results. This improves the previous calibration results from [!cite](Simon2025).
```

Comment:
```suggestion
The results presented here are updated results from those presented in [!cite](Simon2025). First, the initial time step was reduced from dt=60 s in [!cite](Simon2025) to dt=1 s in the current case. This slightly affects the results for both the immediate and delayed injection cases. However, the results are qualitatively unchanged and conclusions remain valid. Second, the calibration approach was updated since [!cite](Simon2025) with an updated multi-objective function, and new results. This improves the previous calibration results from [!cite](Simon2025).
```